installation-and-deployment/running-with-pyenv.md

```diff
@@ -3,7 +3,7 @@
 If you use `pyenv` to manage multiple versions of Python, you can run Aqueduct as a part of your `pyenv` version management.&#x20;
 
 * Aqueduct supports Python versions 3.7-3.10, so please use one of those versions of Python when installing Aqueduct. **Aqueduct will not work with Python versions 3.6 or older.**
-* Note that because the Aqueduct backend server spins off separate Python processes on a periodic basis, you will need to configure your `pyenv` versions in one of two ways:
+* Note that because the Aqueduct backend server spins off separate Python processes on a periodic basis, you will need to configure your `pyenv` versions at the terminal where you ran `aqueduct start` in one of two ways:
```

Comment:
Hm, I'm actually a little confused about this @Fanjia-Yan. Isn't the goal of `pyenv` to persist across multiple sessions and in a directory-based fashion? So if I set `pyenv local` in `$HOME/.aqueduct`, will that not persist across terminal sessions?

Reply:
Sorry for the confusion, after I re-read the `pyenv` documentation. I haven't fully setup the terminal for automatically initializing the `pyenv`, which causes the version mismatch across terminal sessions. 
Don't think this edition is necessary now. Thanks for pointing out!

Reply:
Got it! So everything should work as is, then, correct? If so, feel free to close this PR and mark the task as done. Thanks @Fanjia-Yan!